clap_derive/src/derives/attrs.rs

```diff
@@ -624,16 +624,13 @@ impl Attrs {
     pub fn has_explicit_methods(&self) -> bool {
         self.methods
             .iter()
-            .any(|m| m.name != "help" && m.name != "long_help")
+            .any(|m| m.name != "about" && m.name != "long_about")
```

Comment:
Because if this:
https://github.com/clap-rs/clap/blob/6910a93ee70b5992912363373e766cccb50474cb/clap_derive/src/derives/attrs.rs#L460-L465


```rust
structopt Opt {
    #[clap(subcommand, some_method = expr)]
    field: SubCmd
}
```

`#[subcommand]` is always used solo, if there's another method on top of the field, this is obviously a misplaced method. That code ensures that the snippet above will trigger an error so programmer will be warned about misuse.

But doc comments are also methods, in fact, and doc comments are allowed here. Yes, they don't mean anything (doc comment for subcommand are supposed to be placed on each variant separately, not here) and they are just ignored. Probably we should have changed it, but, at the time of the last git blame, there was already a lot code had been written relying on doc comments, and forbidding those comments would break a lot of code. So we just said "so be it" and allowed `help` and `long_help` (this is how doc commands fly under the hood) there. 

We an change it in clap_derive though.